src/ResourceManager/PowerBIDedicated/Commands.PowerBIDedicated/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Commands.PowerBIDedicated")]
+[assembly: AssemblyDescription("")]
```

Comment:
@selasagi please fill in as much information as you can. You can use the [`AssemblyInfo.cs`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Compute/Commands.Compute/Properties/AssemblyInfo.cs#L20-L23) file in Compute's project as an example

Reply:
done